app/templates/experiment.hbs

```diff
@@ -1,10 +0,0 @@
-<h4> TODO: breadcrumbs </h4>
```

Comment:
This file became `app/templates/experiments/info/index.hbs`. The session stuff at the top was just example code from Andy.